tests/unit/services/loading-test.ts

```diff
@@ -1,49 +1,145 @@
 import { module, test } from 'qunit';
 import { setupTest } from 'ember-qunit';
 import { timeout } from 'ember-concurrency';
+import LoadingService from 'ember-loading/services/loading';
+import { defer } from 'rsvp';
 
 module('Unit | Service | loading', function(hooks) {
   setupTest(hooks);
 
-  test('switching on/off is behaving properly', function(assert) {
-    let service = this.owner.lookup('service:loading');
+  test('can run async jobs', async function(assert) {
+    let deferred = defer();
+    let called = false;
+    let job = () => {
+      called = true;
```

Comment:
This could be `called++`. In the end of the test we could do `assert.equal(called, 1)`.

Reply:
👍